README.md

```diff
@@ -0,0 +1,18 @@
+# OpenNumismat
+
+OpenNumismat is a software for management custom coin collection catalog.
+
+This repo hosts the flatpak wrapper for [OpenNumismat](https://opennumismat.github.io/), available at [Flathub](https://flathub.org/apps/io.github.opennumismat.OpenNumismat).
+
+```sh
+flatpak install flathub io.github.opennumismat.OpenNumismat
+flatpak run org.opennumismat.OpenNumismat
+```
+
+### Wayland
+
+This package uses the flags to run on Wayland. To opt-out it run:
```

Comment:
no user will ever see this. Also if this is really needed then it should be the default.

Reply:
This is an alternative. Made a note to myself